plugins/permission-node/src/integration/createPermissionIntegrationRouter.test.ts

```diff
@@ -95,14 +96,22 @@ describe('createPermissionIntegrationRouter', () => {
     ])('returns 200/ALLOW when criteria match (case %#)', async conditions => {
       const response = await request(app)
         .post('/.well-known/backstage/permissions/apply-conditions')
-        .send({
-          resourceRef: 'default:test/resource',
-          resourceType: 'test-resource',
-          conditions,
-        });
+        .send([
+          {
+            id: '123',
+            resourceRef: 'default:test/resource',
+            resourceType: 'test-resource',
+            conditions,
+          },
+        ]);
 
       expect(response.status).toEqual(200);
-      expect(response.body).toEqual({ result: AuthorizeResult.ALLOW });
+      expect(response.body).toEqual([
```

Comment:
I'd say it's best to keep an object at the root of this response, and nest the result array within that. With an array at the top level you're locked out of adding any additional metadata about the contents of the array, so it's usually something you regret down the line. Just a `{"items":[...]}` or something like that

Reply:
Makes sense! Will circle back on this once I've explored @freben's suggestion above.

Reply:
Done in 069265bfe8. Note that this probably implies a similar change in the request and response objects used for the /authorize call itself - I'll make that change in a separate PR.